test/NuGet.Core.Tests/NuGet.Build.Tasks.Pack.Test/PackTaskLogicTests.cs

```diff
@@ -64,6 +65,400 @@ public void PackTaskLogic_ProducesBasicPackage()
             }
         }
 
+        [Fact]
```

Comment:
These tests look great. 

Too many scenarios to truly determine how complete they are, but they look fine to me.